components/TextArea/textarea.test.tsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Textarea } from './textarea';
+
+describe("Textarea", () => {
+    it("Renders the Textarea correctly", () => {
+        render(<Textarea />)
+        const textarea = screen.getByRole("textbox")
+        expect(textarea).toBeInTheDocument()
+    })
+    it("Renders the Textarea with custom className", ()=>{
+        render(<Textarea className="custom-cn" />)
+        const textarea = screen.getByRole("textbox")
+        expect(textarea).toHaveClass("custom-cn")
+    })
+    it("Renders the Textarea with props", ()=>{
+        render(
+            <Textarea 
+                placeholder="test-placeholder"
+                name="test-name"
+                defaultValue="test-value"
+            />)
+        const textarea = screen.getByRole("textbox")
+        expect(textarea).toHaveAttribute("placeholder", "test-placeholder")
+        expect(textarea).toHaveAttribute("name", "test-name")
+        expect(textarea).toHaveValue("test-value")
+    })
+    it("Should be able to type into Textarea", () => {
+        render(<Textarea />)
+        const textareaElement = screen.getByRole<HTMLTextAreaElement>("textbox")
+        fireEvent.change(textareaElement, {target: { value: "testing the textarea element" }})
```

Comment:
refactor (blocking): use `userEvent.type` to emulate a user entering text in the text area. This will also fire any other events (if any wired up) along with the onChange event.

You'll also need to install `@testing-library/user-event` and import `userEvent`.

```
pnpm install @testing-library/user-event
```

```typescript
import userEvent from '@testing-library/user-event';
```


```suggestion
    it("Should be able to type into Textarea", async () => {
        render(<Textarea />)
        const textareaElement = screen.getByRole<HTMLTextAreaElement>("textbox")
        await userEvent.type(textareaElement, 'testing the textarea element');
```

Giving you a ping @shashilo as this introduces a new package to the project.

Reply:
Got it - Should installing the new package be completed in a separate ticket, or can I install/import within this ticket? @shashilo

Reply:
@nickytonline @shashilo - fireEvent is now changed to userEvent in the text area test